deposit/src/main/java/edu/unc/lib/deposit/work/DepositMessageHandler.java

```diff
@@ -1,94 +0,0 @@
-/**
- * Copyright 2008 The University of North Carolina at Chapel Hill
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package edu.unc.lib.deposit.work;
```

Comment:
Yes, most likely, and there wasn't anything to submit requests for indexing to in the fcrepo4 branch at this point